src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -25,16 +25,17 @@ private RecipeHandler()
      * Initialize all recipes for minecolonies.
      *
      * @param enableInDevelopmentFeatures if we want development recipes.
-     * @param supplyChests                if we want supply chests or direct town hall crafting.
+     * @param supplyChests                if we want supply chests or direct
+     *                                    town hall crafting.
      */
     public static void init(final boolean enableInDevelopmentFeatures, final boolean supplyChests)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "enableInDevelopmentFeatures" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "supplyChests" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2301)